k8sclient/k8sclient.go

```diff
@@ -81,9 +81,21 @@ func setKubeClientInfo(c *clientInfo, client KubeClient, k8sArgs *types.K8sArgs)
 	c.Podname = string(k8sArgs.K8S_POD_NAME)
 }
 
-func SetNetworkStatus(client KubeClient, k8sArgs *types.K8sArgs, netStatus []*types.NetworkStatus) error {
-
+func SetNetworkStatus(client KubeClient, k8sArgs *types.K8sArgs, netStatus []*types.NetworkStatus, conf *types.NetConf) error {
 	logging.Debugf("SetNetworkStatus: %v, %v, %v", client, k8sArgs, netStatus)
+
+	client, err := GetK8sClient(conf.Kubeconfig, client)
+	if err != nil {
+		return err
```

Comment:
Minor comment, but could you wrap err with logigng.Errorf(), for ease of troubleshooting, to understand which function failed to call GetK8sClient()?

Example:
```
return logging.Errorf("SetNetworkStatus: %v", err)
```

Reply:
Added.